### PR TITLE
Log becoming multicellular in the timeline

### LINKED ISSUE #3154: Log becoming multicellular in the timeline
so that it leaves a trace visible afterwards. Probably should have its own icon for this event.

### PR DESCRIPTION
**Pull request by ender-titan1**
**Brief Description of What This PR Does**

This PR logs becoming multicellular in the timeline

**Related Issues**

Closes #3154

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [X] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [ ] Initial code review passed (this and further items should not be checked by the PR author)
- [ ] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

File: assets/models/easter_eggs/GooglyEyeCellFlee.anim
Comment (hhyyrylainen) [inline assets/models/easter_eggs/GooglyEyeCellFlee.anim:1]: Looks like you allowed Godot to do these changes I've always kept reverting... Does this model and the two animations still work?

File: src/microbe_stage/MicrobeStage.cs [hunk 1]
```diff
@@ -367,6 +367,15 @@ public void MoveToMulticellular()
             return;
         }
 
+        // Log becoming multicellular in the timeline
+        GameWorld.LogEvent(
+            new LocalizedString("TIMELINE_PLAYER_BECAME_MULTICELLULAR", Player.Species.FormattedName),
```
Comment (hhyyrylainen) [inline src/microbe_stage/MicrobeStage.cs:372]: In the future becoming multicellular is not exclusive to the player, so I think the translation key should be "TIMELINE_SPECIES_BECAME_MULTICELLULAR" (and the text needs to be adjusted as well if it makes explicit reference to the player).

**Review by hhyyrylainen — CHANGES_REQUESTED** [review]

File: assets/models/easter_eggs/GooglyEyeCellFlee.anim
Comment (ender-titan1) [reply assets/models/easter_eggs/GooglyEyeCellFlee.anim:1]: The slime jet animation appears to work for sure, as for the other ones, I don't really know how to check.

**Review by hhyyrylainen — APPROVED** [review]
This seems fine, though right now the species becoming multicellular is logged as 100 Myr earlier in the global timeline as the local one. Someone might notice this at some point and report it as a bug...

**Pull request merged**